src/components/input-dropdown/input-dropdown.component.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('InputDropdownComponent', () => {
         spyOn(component.filterChange, 'emit');
         spyOn(component.filterInputElement.nativeElement, 'focus');
 
-        component.resetFilter(new MouseEvent('click'));
+        component.resetFilter();
```

Comment:
Probably this test should be modified, or a new test created, to verify that selecting an item clears the filter.

Reply:
I am not sure about the second build link. How can I create a build for ux-aspecs-micro-focus, which consumes my unmerged changes from the UXAspects repository?

Reply:
If you create a branch with the same name and create a build for that, the build will pick up this build artifact as the dependency.